src/test/resources/com/puppycrawl/tools/checkstyle/checks/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -102,4 +102,17 @@ void foo() {
 	private String notAUnicodeEscaped2 = "\\\\u1234";
 
 	private String onlyEscaped = "\\\u1234";
+
+	private String controlC0 = ""
+			+ "\u0000\u0001\u0002\u0003\u0004\u0005\u0006\u0007\u0008\u0009\n\u000b\u000c\r\u000e\u000f"
```

Comment:
@Luolc Is there anyway to use the unicode equivalent without a compiler error?
The point of this check and this test is verify unicode ranges. We should have all in the range if possible.
If it is not possible at all in the source code, then we should probably add a comment so it is clear why they are not in the list in their specific position.